k2/python/csrc/torch.h

```diff
@@ -0,0 +1,18 @@
+/**
+ * @brief python wrappers for PyTorch.
```

Comment:
we need to make clear that these are wrappers for k2 for use in PyTorch.  Also variable names should make this obvious..